package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "coding-miproyectobase",
   "private": true,
+  "version": "0.0.3",
```

Biome (1.9.4)

[error] The key version was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)